database/seeders/AddHasAdressesSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Address;
+use App\Models\Area;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class AddhasAdressesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $addressesAreasIds=Address::select('areaId')->get();
+
+        DB::table('areas')
+        ->whereExists(function ($query){
```

Comment:
Try checking other methods instead of using 'whereExists' as it won't be optimized in some cases